core/src/main/java/io/confluent/rest/RestConfig.java

```diff
@@ -1125,13 +1127,18 @@ private static ConfigDef incompleteBaseConfigDef() {
 
   private static Time defaultTime = Time.SYSTEM;
 
-  public RestConfig(ConfigDef definition, Map<?, ?> originals) {
-    super(definition, originals);
+  public RestConfig(ConfigDef definition, Map<?, ?> originals, boolean doLog) {
```

Comment:
why not use the config `CONFIGS_LOGGING_ENABLED_PROP` as the original PR proposed?

Using the config has 1 pro, that it can be modified at run-time to toggle logging on/off.

Reply:
CONFIGS_LOGGING_ENABLED_PROP is internal (ce-kafka only), so it will be used in upstream of this project (ce-kafka-rest and ce-kafka-http-server).